formats.go

```diff
@@ -90,12 +90,15 @@ func writeCertificateFile(filename string, data map[string]interface{}) error {
 }
 
 func writeCertificateBundleFile(filename string, data map[string]interface{}) error {
-	certificateFile := fmt.Sprintf("%s.crt", filename)
+	bundleFile := fmt.Sprintf("%s.crt", filename)
+	keyFile := fmt.Sprintf("%s.key", filename)
 	caFile := fmt.Sprintf("%s.ca", filename)
-	certificate := fmt.Sprintf("%s\n\n%s", data["certificate"], data["private_key"])
+
+	bundle := fmt.Sprintf("%s\n\n%s", data["certificate"], data["issuing_ca"])
```

Comment:
Probably want to add a `\n` at the end: `fmt.Sprintf("%s\n\n%s\n"`

Same goes for other files.